pages/api/reset-password.js

```diff
@@ -0,0 +1,34 @@
+import withContainer from "../../src/middleware/withContainer";
+
+export default withContainer(async ({ body, method }, res, { container }) => {
+  if (method !== "POST") {
```

Comment:
See comment on Azlina's PR: https://github.com/madetech/nhs-virtual-visit/pull/743#discussion_r557279999, there is another file that can be updated too

Reply:
Refactored to use helper methods